calcom/cal.com
# fix: Removing 2fa text that appears twice

## What does this PR do?

Fixes #9761 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

In the DisableTwoFactorModal.tsx file, there was a label component and TwoFactor component both displaying the same message so I removed the Label one.

Not sure if this is the way to do it not. Feel free to review or provide any sort of feedback. My first time going through this repo.

## How should this be tested?

Try to disable 2FA after enabling it. It used to appear now but now it only appears once.
